dropshot/tests/test_openapi_fuller.json

```diff
@@ -414,6 +510,25 @@
           "x"
         ]
       },
+      "Error": {
+        "description": "Error information from a response.",
+        "type": "object",
+        "properties": {
+          "error_code": {
+            "type": "string"
+          },
+          "message": {
+            "type": "string"
+          },
+          "request_id": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "message",
+          "request_id"
+        ]
```

Comment:
🤦 yes `error_code` can be left out. Never mind. Though actually I have a question — does not required mean it can be given a value of `null`, or only that it can be left out?